pom.xml

```diff
@@ -144,11 +157,25 @@
             <scope>provided</scope>
         </dependency>
 
+        <dependency>
+            <groupId>net.essentialsx</groupId>
+            <artifactId>EssentialsX</artifactId>
+            <version>2.19.1-SNAPSHOT</version>
+            <scope>provided</scope>
+        </dependency>
+
         <dependency>
             <groupId>com.github.Gypopo</groupId>
             <artifactId>EconomyShopGUI-API</artifactId>
             <version>1.7.1</version>
             <scope>provided</scope>
         </dependency>
+
+        <dependency>
+            <groupId>CMI-API</groupId>
+            <artifactId>CMI-API</artifactId>
+            <version>9.7.4.1</version>
+            <scope>provided</scope>
+        </dependency>
```

Comment:
Oh god that's gonna break at some point ✝️ 
We almost dropped CMI support from the core because it is such a royal fuck-up